openmls/src/group/mls_group/mod.rs

```diff
@@ -471,20 +475,6 @@ impl MlsGroup {
     pub(crate) fn group(&self) -> &CoreGroup {
         &self.group
     }
-
-    /// Removes a specific proposal from the store.
-    pub fn remove_pending_proposal<Storage: StorageProvider>(
```

Comment:
Moving this function looks unrelated? Is this intentional?

Reply:
It is now unrelated (I touched it during the implementation). Moving it was however intentional because it became public at some point and was in the wrong `impl` block.